jquery.timepicker.d.ts

```diff
@@ -15,7 +15,7 @@ declare namespace Jt.Timepicker {
          * Override where the dropdown is appended.
          * Takes either a string to use as a selector, a function that gets passed the clicked input element as argument or a jquery object to use directly.
          */
-        appendTo?: string | ((clickedElement) => string);
+        appendTo?: string | ((clickedElement: JQuery) => string);
```

Comment:
Would this make sense? I've never really used Typescript.

```suggestion
        appendTo?: string | JQuery | ((clickedElement: JQuery) => JQuery);
```

Reply:
Looks good to me and I also tested it in TypeScript and it reports no error!

Reply:
Ok great! Can you add these changes to your branch, and I'll go ahead and merge?

Reply:
You already changed it my branch as far as I can tell.